src/parser.rs

```diff
@@ -1176,7 +1176,10 @@ impl Parser {
                         return parser_err!(format!("No value parser for keyword {}", k.keyword));
                     }
                 },
-                Token::Number(ref n) => Ok(Value::Number(n.to_string())),
+                Token::Number(ref n) => match n.parse() {
```

Comment:
I was confused by this at first, as the IDE suggested that `parse()` returns a `String`. Perhaps add a comment that this is for feature=bigdecimal?

Reply:
Sure thing, done!